GMG/Scenes/Home/HomeView.swift

```diff
@@ -117,19 +122,19 @@ struct HomeView: View {
                                 size: 42
                             )
 
-                            VStack(alignment: .leading) {
+                            VStack(alignment: .leading, spacing: 8) {
```

Comment:
Spacing에 있는 간격은 사용 부탁드립니다!

```suggestion
                            VStack(alignment: .leading, spacing: Spacing.xs) {
```